src/Web3Provider/BrowserWeb3Provider.ts

```diff
@@ -29,28 +32,42 @@ export class BrowserWeb3Provider implements IWeb3Provider {
   public async connectWallet() {
     const { default: Torus } = await import('@toruslabs/torus-embed')
     const providerOptions: IProviderOptions = {
-      torus: this.walletSetting?.providers?.torus
-        ? {
-            package: Torus, // required
-            display: this.walletSetting?.providers?.torus?.display,
-            options: {
-              config: this.walletSetting?.providers?.torus?.options,
-            },
-          }
-        : { package: Torus },
+      walletconnect: {
```

Comment:
To make this work on the server-side as well, please refer to the following for an implementation.

https://github.com/KyuzanInc/mint-sdk-js/pull/135/files#diff-0371edcfe82aee0567c20b68b2c0e4d555f65a170880ec4dbd88d53f0e94f275L30

Reply:
@tomonari-t -san. Do you mean that WalletSeting must have a default `RPC` value?

https://github.com/KyuzanInc/mint-sdk-js/pull/135/files#diff-0371edcfe82aee0567c20b68b2c0e4d555f65a170880ec4dbd88d53f0e94f275R47-R50

Reply:
@longlh1997 
No.
I mean that let's use `dynamic import`
```typescript
const { default: Torus } = await import('@toruslabs/torus-embed')
```

Reply:
@tomonari-t -san. I have updated it. Please check it. Thanks